Makefile

```diff
@@ -135,9 +141,9 @@ verify: .init .generate_files
 	@# we have just one container for everything and not one container per
 	@# file.  The $(subst) removes the "-t" flag from the Docker cmd.
 	@echo for i in \`find $(TOP_SRC_DIRS) -name \*.go \| grep -v zz\`\; do \
-	  golint --set_exit_status \$$i \&\& \
+	  golint --set_exit_status \$$i \; \
```

Comment:
I'm reminded of Jurassic Park when Jeff says something along the lines of: they were so fascinated by whether they could, that they never stopped to ask if they should.
:-)